src/web/mjs/connectors/KissLove.mjs

```diff
@@ -0,0 +1,13 @@
+import FlatManga from './templates/FlatManga.mjs';
+
+export default class LHScan extends FlatManga {
+
+    constructor() {
+        super();
+        super.id = 'kisslove';
+        super.label = 'KissLove';
+        this.tags = [ 'manga', 'raw', 'japanese' ];
+        this.url = 'https://kisslove.net/';
```

Comment:
May remove trailing slash

Reply:
omg why i keep forgetting to remove slash. fixed that alr, sry